FBSDKMessengerShareKit/FBSDKMessengerShareKit.xcodeproj/project.pbxproj

```diff
@@ -534,41 +534,49 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 9CAE101E1B15355200163BDA /* FacebookSDK-LogicTests.xcconfig */;
 			buildSettings = {
+				PRODUCT_BUNDLE_IDENTIFIER = "com.facebook.sdk.$(PRODUCT_NAME:rfc1034identifier)";
 			};
 			name = Debug;
 		};
 		9CAE101C1B15353900163BDA /* Release */ = {
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 9CAE101E1B15355200163BDA /* FacebookSDK-LogicTests.xcconfig */;
 			buildSettings = {
+				PRODUCT_BUNDLE_IDENTIFIER = "com.facebook.sdk.$(PRODUCT_NAME:rfc1034identifier)";
 			};
 			name = Release;
 		};
 		9D0558B71A797E7B009B7108 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 23AF544A1A8AB66C001D9871 /* FacebookSDK-Project-Debug.xcconfig */;
 			buildSettings = {
+				ENABLE_TESTABILITY = YES;
```

Comment:
similarly, move this and GCC_NO_COMMON_BLOCKS to FBSDKMessengerShareKit/Configurations/FacebookSDK-Project-Debug.xcconfig